[2025-06-19T18:02:40.664+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-19T18:02:40.746+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_classify_v1.check_file_exists_branch manual__2025-06-19T09:02:28.125954+00:00 [queued]>
[2025-06-19T18:02:40.786+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_classify_v1.check_file_exists_branch manual__2025-06-19T09:02:28.125954+00:00 [queued]>
[2025-06-19T18:02:40.788+0900] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-19T18:02:40.848+0900] {taskinstance.py:2890} INFO - Executing <Task(_BranchPythonDecoratedOperator): check_file_exists_branch> on 2025-06-19 09:02:28.125954+00:00
[2025-06-19T18:02:40.889+0900] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8896) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-19T18:02:40.896+0900] {standard_task_runner.py:72} INFO - Started process 8913 to run task
[2025-06-19T18:02:40.899+0900] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_classify_v1', 'check_file_exists_branch', 'manual__2025-06-19T09:02:28.125954+00:00', '--job-id', '1672', '--raw', '--subdir', 'DAGS_FOLDER/img_classify.py', '--cfg-path', '/tmp/tmpnlpfkaf8']
[2025-06-19T18:02:40.912+0900] {standard_task_runner.py:105} INFO - Job 1672: Subtask check_file_exists_branch
[2025-06-19T18:02:41.090+0900] {task_command.py:467} INFO - Running <TaskInstance: image_classify_v1.check_file_exists_branch manual__2025-06-19T09:02:28.125954+00:00 [running]> on host f6d4228a9eb8
[2025-06-19T18:02:41.437+0900] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_classify_v1' AIRFLOW_CTX_TASK_ID='check_file_exists_branch' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T09:02:28.125954+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T09:02:28.125954+00:00'
[2025-06-19T18:02:41.443+0900] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-19T18:02:41.462+0900] {python.py:240} INFO - Done. Returned value was: get_file_info_list_task
[2025-06-19T18:02:41.465+0900] {branch.py:38} INFO - Branch into get_file_info_list_task
[2025-06-19T18:02:41.469+0900] {skipmixin.py:233} INFO - Following branch ('get_file_info_list_task',)
[2025-06-19T18:02:41.506+0900] {skipmixin.py:281} INFO - Skipping tasks [('end_task', -1)]
[2025-06-19T18:02:41.635+0900] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-19T18:02:41.637+0900] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_classify_v1, task_id=check_file_exists_branch, run_id=manual__2025-06-19T09:02:28.125954+00:00, execution_date=20250619T090228, start_date=20250619T090240, end_date=20250619T090241
[2025-06-19T18:02:41.783+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-19T18:02:41.939+0900] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-19T18:02:41.948+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
